client/src/test/kotlin/io/provenance/client/PbClientTest.kt

```diff
@@ -33,17 +37,38 @@ class PbClientTest {
             )
         }
     }
+    @Test
```

Comment:
I was hoping to add more testing, but I was not able to easily mock the grpc stubs. Instead a did a few local functional tests. Depending on adoption it may make sense to tackle setting up integration tests fo this client.